azure-pipelines-integration.yml

```diff
@@ -68,6 +70,15 @@ jobs:
       continueOnError: true
       condition: not(succeeded())
 
+    - task: PublishBuildArtifacts@1
+      displayName: Publish Secondary Logs
+      inputs:
+        PathtoPublish: '$(Build.SourcesDirectory)\artifacts\log2\$(_configuration)'
```

Comment:
Don't think we should be craeting a log2 directory. That complicates our infrastructure here. Should be focusing on a single log dir.

Reply:
This restored the ability to collect xunit.console.x86 logs following a hang. Test hangs cannot be diagnosed without this file.

I'm happy to remove this separation once we resolve the underlying issue, but I believe the inability to diagnose test failures is the more acute issue.

Reply:
➡️ Submitted #35726 to revert this after the Azure Pipelines artifact limitations are fixed.

Reply:
What limitation in azure are you referring to? Our investigation with them revealed that it's almost certainly our problem, not there's.

Reply:
Before this change, only 1 of 8 hung builds included the dump file (0/1 in [20190514.12](https://dev.azure.com/dnceng/public/_build/results?buildId=188711), 0/3 in [20190509.23](https://dev.azure.com/dnceng/public/_build/results?buildId=184946), and 1/4 in [20190509.7](https://dev.azure.com/dnceng/public/_build/results?buildId=184519)). After this change, both hung builds in [20190514.34](https://dev.azure.com/dnceng/public/_build/results?buildId=189228) included this file.

Reply:
That just continues to confirm the problem is on our end, not Azure's. If this is an Azure bug then we need to have data to give them about it. Specifically, as they requested in the  email, list the contents of the directory showing we wrote the file and they didn't upload it.